<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1132</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>1121</width>
      <height>541</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Задание №1</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>32</x>
        <y>190</y>
        <width>137</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>176</x>
        <y>190</y>
        <width>137</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>190</y>
        <width>137</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>190</y>
        <width>211</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Найти корни уравнения</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>230</y>
        <width>441</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Ответ:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_task1">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>581</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Написать программу, которая будет находить корень квадратного уравнения второй степени,&lt;/p&gt;&lt;p&gt;входные данные: a, b, c. Если нахождение корня невозможно, выводить сообщение в поле&lt;/p&gt;&lt;p&gt;результата (или выводить результат в виде комплексного числа). Учитывать, что число a&lt;/p&gt;&lt;p&gt;может быть равно нулю.&lt;/p&gt;&lt;p&gt;Найти корни квадратного уравнения - &lt;span style=&quot; font-weight:600;&quot;&gt;ax&lt;/span&gt;&lt;span style=&quot; font-weight:600; vertical-align:super;&quot;&gt;2&lt;/span&gt;&lt;span style=&quot; font-weight:600;&quot;&gt; + bx + c = 0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>160</y>
        <width>441</width>
        <height>18</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Введите значение a:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Введите значение b:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Введите значение c:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Задание №2</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>410</y>
        <width>541</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Ответ:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>170</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Расчитать длину</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>300</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Задать значение угла в градусах</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_pic">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>521</width>
        <height>291</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>img/tre.PNG</pixmap>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::LinksAccessibleByMouse</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_task1_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>561</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Расчет стороны треугольника по двум сторонам и углом между ними. Дополнительно:&lt;/p&gt;&lt;p&gt;добавить переключатель из QRadioButton для выбора единиц, в каких указан угол: градусах&lt;/p&gt;&lt;p&gt;или радианах. Формулу для расчета можно найти по ссылке:&lt;/p&gt;&lt;p&gt;https://www-formula.ru/2011-10-09-11-08-41.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>150</y>
        <width>131</width>
        <height>251</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="lineEdit_4"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_5"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_6"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_7"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_8"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_9"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>150</y>
        <width>131</width>
        <height>251</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_A_2">
         <property name="text">
          <string>Введите значение a:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_A_3">
         <property name="text">
          <string>Введите значение b:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_A_4">
         <property name="font">
          <font>
           <italic>false</italic>
          </font>
         </property>
         <property name="text">
          <string>Введите значение c:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_A_5">
         <property name="text">
          <string>Введите значение α:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_A_7">
         <property name="font">
          <font>
           <italic>false</italic>
          </font>
         </property>
         <property name="text">
          <string>Введите значение β:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_A_6">
         <property name="text">
          <string>Введите значение y:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Задание №3</string>
     </attribute>
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>231</width>
        <height>141</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>110</y>
        <width>231</width>
        <height>141</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_task1_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>701</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;*Создать программу, разместить на поле виджет QPlainTextEdit. Добавить кнопки: для&lt;/p&gt;&lt;p&gt;добавления фиксированной строки, замены на готовый текст. Разместить QTextEdit. Через&lt;/p&gt;&lt;p&gt;метод setHtml(QString) установите текст произвольного html-кода, например &amp;lt;font color='red'&amp;gt;Hello&amp;lt;/font&amp;gt;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>120</y>
        <width>141</width>
        <height>111</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>Добавить строку</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_4">
         <property name="text">
          <string>Готовый текст</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Задание №4</string>
     </attribute>
     <widget class="QLabel" name="label_task1_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>591</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Получите QTextDocument из виджета QTextEdit (метод document) . С помощью списка&lt;/p&gt;&lt;p&gt;методов и переменных выберите, какие наиболее интересны для редактирования текста.&lt;/p&gt;&lt;p&gt;Ознакомьтесь с работой подсказчика кода Qt Creator ( Ctrl+Space ).&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Введите любой текст:</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>140</y>
        <width>351</width>
        <height>151</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>150</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Изменить шрифт</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1132</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
